### ISSUE TITLE
Subscription panel is not changed after changing plan

### ISSUE THREAD

**Issue by RandomAdversary** [ProtonMail/protonvpn-settings#56]
**Steps to reproduce**
1. Go to /dashboard
2. Notice the info on the subscription panel.
3. Upgrade to different plan
4. Notice the info on the subscription panel.

**Expected behavior**
After step 3, the subscription panel should show the new plan.

**Comment by EpokK** [comment]
We are not receiving update for `Subscription` model, it's an API issue.
